apps/ledger-live-desktop/src/newArch/features/Collectibles/Ordinals/components/RareSats/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { useRareSatsModel } from "./useRareSatsModel";
+import TableContainer from "~/renderer/components/TableContainer";
+import TableHeader from "LLD/features/Collectibles/components/Collection/TableHeader";
+import { TableHeaderTitleKey } from "LLD/features/Collectibles/types/Collection";
+import { Box, Text, Flex } from "@ledgerhq/react-ui";
+import { TableHeader as TableHeaderContainer } from "./TableHeader";
+import RowLayout from "LLD/features/Collectibles/Ordinals/components/RareSats/RowLayout";
+import IconContainer from "LLD/features/Collectibles/components/Collection/TableRow/IconContainer";
+import TokenTitle from "LLD/features/Collectibles/components/Collection/TableRow/TokenTitle";
+import { RareSat } from "LLD/features/Collectibles/types/Ordinals";
+
+type ViewProps = ReturnType<typeof useRareSatsModel>;
+
+const Item = ({
+  icons,
+  name,
+  year,
+  count,
+  utxo_size,
+  isMultipleRow,
+}: RareSat & { isMultipleRow: boolean }) => {
+  const firstColumn = (
+    <Flex columnGap={2}>
+      {icons && <IconContainer icons={Object.values(icons)} />}
+      <TokenTitle tokenName={[name]} complementaryData={count} isLoading={false} />
+    </Flex>
+  );
+  const secondColumn = (
+    <Text variant={"bodyLineHeight"} fontSize={12} color={"neutral.c70"}>
+      {year}
+    </Text>
+  );
+  const thirdColumn = (
+    <Text variant={"bodyLineHeight"} fontSize={12} color={"neutral.c70"}>
+      {utxo_size}
+    </Text>
+  );
+
+  return (
+    <RowLayout
+      isMultipleRow={isMultipleRow}
+      firstColumnElement={firstColumn}
+      secondColumnElement={secondColumn}
+      thirdColumnElement={thirdColumn}
+    />
+  );
+};
+
+function View({ rareSats }: ViewProps) {
+  return (
+    <Box>
+      <TableContainer id="oridinals-inscriptions">
+        <TableHeader titleKey={TableHeaderTitleKey.RareSats} />
+        <TableHeaderContainer />
+        <Flex flexDirection={"column"}>
+          {rareSats.map((rareSatGroup, index) => (
+            <Flex key={index} flexDirection={"column"}>
+              {rareSatGroup.sats.map((rareSat, subIndex) => (
+                <Item
+                  key={subIndex}
```

Comment:
same here , no index in key

Reply:
I change for the utxo_size since they are grouped by uxto_size for this one